platforms/corda-ent/configuration/deploy-network.yaml

```diff
@@ -53,5 +55,14 @@
     loop: "{{ network['organizations'] }}"
     loop_control:
       loop_var: org
+    when: network.type == 'corda-enterprise'
```

Comment:
Is network.type necessary even if the condition is in site.yaml?

Reply:
It isn't. I have changed the code.